roles/php/tasks/main.yml

```diff
@@ -81,10 +83,10 @@
   notify:
     - "restart php72-php-fpm"
 
-- name: Enable php72-php-fpm
-  service:
-    name: php72-php-fpm
-    enabled: yes
+# - name: Enable php72-php-fpm
+#   service:
+#     name: php72-php-fpm
+#     enabled: yes
 
```

Comment:
Why is this task commented?

Reply:
I commented it (Would actually wanted to remove it) but the handler is already enabling and (re)starting the service. So having a 2nd task which only does the enabling is a bit double.

Reply:
Right, removing it should be ok than!

Reply:
Yes, removed it. Thanks! 👍